x-pack/legacy/plugins/integrations_manager/public/hooks/breadcrumbs.tsx

```diff
@@ -0,0 +1,13 @@
+/*
```

Comment:
Can we call the hook files `use_` ?

Reply:
Updated in https://github.com/elastic/kibana/commit/f40c1728bd181096a4066eced6bfbf568df41477